src/mpilib20_init_finalise.F90

```diff
@@ -19,10 +19,13 @@ module mpilib20_init_finalise
      integer         :: process      !> Process id (rank)
      integer         :: n_processes  !> Total number of processes
      type(MPI_Group) :: group        !> Group id
-     integer         :: ierror       !> Error code   
+     integer         :: group_size   !> Processes in group
```

Comment:
I'm not sure we need this. We only create groups to create new communicators, so it might be redunant. I.e. each group will have n_processes. I'll need to go back and look at what I wrote for DL_POLY 

- [ ] Add comment may be superfluous - review

Reply:
Keep